eth/tracers/js/internal/tracers/rosetta.js

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
Can we remove this? Rosetta switched to supplying this itself, so it was removed from coreth.